tests/operators/test_snowflake_decorator.py

```diff
@@ -142,7 +152,7 @@ def tearDown(self):
     def test_snowflake_query(self):
         @aql.transform
         def sample_snow(input_table: Table):
-            return f"SELECT * FROM {input_table.table_name} LIMIT 10"
+            return "SELECT * FROM {input_table} LIMIT 10"
```

Comment:
Why don't we need the `f-string` anymore?

Reply:
@tatiana these were never supposed to be f-strings and by identifying them as f-strings users are actually opening themselves up to SQL injections. This raises a huge question over whether our current system looks too much like f-strings and therefore might promote bad habits.